package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "binance",
-  "version": "2.11.5",
+  "version": "2.11.6",
```

```suggestion
  "version": "2.12.0",
```

worth changing to a minor release in my opinion. No breaking changes but still a lot of changes/addition.